Space-X Defense Program/Space-X Defense Program/GameScene.swift

```diff
@@ -11,63 +11,87 @@ import GameplayKit
 
 class GameScene: SKScene {
     
-    //hello world
+    private let background = SKSpriteNode(imageNamed: "background")
+    private let player = Player()
+    let projectiles = Projectile()
     
     private var label : SKLabelNode?
     private var spinnyNode : SKShapeNode?
     
+    var gameTimer: Timer!                     //timer for repeating functions
+    
     override func didMove(to view: SKView) {
         
+        backgroundColor = SKColor.black       //defaulted colored to black for the background
+        background.zPosition = -1             //depth of background, -1 makes it go behind other 2D elements
+        background.position = CGPoint(x: size.width / 2, y: size.height / 2 )          //centering the background images
+        background.setScale(1.43)             //Maximizing the pixel resolution for the 1080p
+        addChild(background)                  //adding the background to the scene
+        
+        addChild(player)                      //adding the spaceship to the scene
+        player.position = CGPoint(x: 220 , y: size.height/2)              //positioning the spaceship to spawn on the centre of the leftmost corner
+        player.setScale(0.6)                  //scale the spaceship down
+        
+        //addChild(projectiles)
+        
+        
+        
         // Get label node from scene and store it for use later
-        self.label = self.childNode(withName: "//helloLabel") as? SKLabelNode
+        /*self.label = self.childNode(withName: "//helloLabel") as? SKLabelNode
         if let label = self.label {
             label.alpha = 0.0
             label.run(SKAction.fadeIn(withDuration: 2.0))
-        }
+        }*/
         
         // Create shape node to use during mouse interaction
-        let w = (self.size.width + self.size.height) * 0.05
-        self.spinnyNode = SKShapeNode.init(rectOf: CGSize.init(width: w, height: w), cornerRadius: w * 0.3)
+        let w = (self.size.width + self.size.height) * 0.02
```

Comment:
Should probably make a new class to handle the details of this SKShapeNode, its cluttering up your GameScene code.